ComplexQueriesAndProcedures/ListBorrowedBooks.sql

```diff
@@ -0,0 +1,5 @@
+declare  @BorrowerID int = 22
+SELECT b.title, b.author,b.isbn, l.dateborrowed, l.duedate, l.datereturned
```

Comment:
you could use 
```suggestion
SELECT *
```
when you show all columns and don't override the name of the column